pkg/authorization/controller/defaultrolebindings/defaultrolebindings_test.go

```diff
@@ -0,0 +1,119 @@
+package defaultrolebindings
+
+import (
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	kubeclientfake "k8s.io/client-go/kubernetes/fake"
+	corelisters "k8s.io/client-go/listers/core/v1"
+	rbaclisters "k8s.io/client-go/listers/rbac/v1"
+	clienttesting "k8s.io/client-go/testing"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/kubernetes/pkg/controller"
+)
+
+func TestSync(t *testing.T) {
```

Comment:
No tests for fixup of deletions?

Reply:
> No tests for fixup of deletions?

The way I wrote the controller, that case looks identical to any other, "I don't have given rolebinding"